Automattic/jetpack
# Jetpack Scan Visibility i1 : Show jetpack scan menu item on Atomic sites that defaults to wp-admin view

Fixes https://github.com/Automattic/dotcom-forge/issues/7556

## Proposed changes:

We want to expose jetpack scan history to Atomic sites that is using the default `wp-admin` interface (Untangling Classic View).  This scan link redirects to jetpack cloud and shows only the history page of scan done.

* Check if site is atomic and on `wp-admin` and show scan link 

![image](https://github.com/Automattic/jetpack/assets/47489215/082ea941-8a88-4442-8f98-7fa158bdd31d)


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Using the Jetpack Beta Tester plugin, apply branch to your atomic site
* Change site to `wp-admin` interface at `/settings/general/atomic_site`
* Go jetpack menu and verify scan link is shown
* Click link and verify it redirects to jetpack cloud and show scan history
